docs/goals.md

```diff
@@ -33,17 +31,17 @@ The design goals are listed below.
 
 1.  A formal definition of the canonical data model for representing
     localizable _dynamic message strings_.
+    _Note: not part of the LDML45 release._
```

Comment:
Why not? It's right here: https://github.com/unicode-org/message-format-wg/tree/main/spec/data-model

```suggestion
```